ParkenDD/Colors.swift

```diff
@@ -22,20 +22,34 @@ struct Colors {
 	*/
 	static func colorBasedOnPercentage(_ percentage: Double, emptyLots: Int) -> UIColor {
 
-		let hue = 1 - (percentage * 0.3 + 0.7) // I want to limit this to the colors between 0 and 0.3
+        let lotDensityColor: UIColor = {
+            switch percentage {
+            case 0.6...1:
+                return UIColor(rgba: "#006a39")
```

Comment:
Create a UIColor extension with static variables like `redColor` instead of hardcoding them here.

Also, don't use `redColor`, use an explicit variable name.

Reply:
Good idea, that made it much more readable.